README.md

```diff
@@ -54,10 +54,16 @@ The following configuration options are shared by all Station commands:
   - Windows: `%TEMP%/Filecoin Station Core`
 - `$STATE_ROOT` _(string; optional)_: Station stores logs and module state in
   this directory. Defaults to
+
   - Linux: `${XDG_STATE_HOME:-~/.local/state}/filecoin-station-core`
   - macOS: `~/Library/Application Support/app.filstation.core`
   - Windows: `%LOCALAPPDATA%/Filecoin Station Core`
 
+  **IMPORTANT:** State files must be stored in a directory that's local to the
```

Comment:
```suggestion
  **IMPORTANT:** The`$STATE_ROOT` directory must be local to the
```

Reply:
As a reader I wouldn't be certain whether by "State files" we mean that directory